acoular/fprocess.py

```diff
@@ -0,0 +1,367 @@
+# ------------------------------------------------------------------------------
+# Copyright (c) Acoular Development Team.
+# ------------------------------------------------------------------------------
+"""Implements blockwise processing methods in the frequency domain.
+
+.. autosummary::
+    :toctree: generated/
+
+    RFFT
+    IRFFT
+    AutoPowerSpectra
+    CrossPowerSpectra
+    FFTSpectra
+"""
+
+import multiprocessing
+from warnings import warn
+
+import numpy as np
+from scipy import fft
+from traits.api import Bool, CArray, Either, Instance, Int, Property, Trait, cached_property
+
+from .base import SamplesGenerator, SpectraGenerator, SpectraOut, TimeOut
+from .fastFuncs import calcCSM
+from .internal import digest
+from .spectra import BaseSpectra
+from .tools.utils import SamplesBuffer
+
+CPU_COUNT = multiprocessing.cpu_count()
```

Comment:
This is just the number of (logical) CPUs in the system, not what we might want the FFT to use. What about overcommittment?

Reply:
Based on our experience regarding unwanted overcommittment, I decided to use a Union trait and set the default value to None (which is the scipy. FFT default value)). This means the number of workers is now settable and should only be used purposefully.